guides/common/modules/proc_cloning-a-role.adoc

```diff
@@ -12,3 +13,21 @@ Use the {ProjectWebUI} to clone a role.
 . Click the name of the cloned role and navigate to *Filters*.
 . Edit the permissions as required.
 . Click *Submit* to save your new role.
+
+.Creating an organization-specific manager role
```

Comment:
This is not new, just moved from elsewhere (I turned the original procedure into an example).